packages/dds/tree/src/test/feature-libraries/objectForest.spec.ts

```diff
@@ -111,7 +111,7 @@ describe("object-forest", () => {
 		);
 		const cursor = forest.allocateCursor();
 		forest.moveCursorToPath(undefined, cursor);
-		assert.deepEqual(cursor.getFieldKey(), cursorForMapTreeNode(forest.roots).getFieldKey());
+		assert.deepEqual(cursor.fieldIndex, cursorForMapTreeNode(forest.roots).fieldIndex);
```

Comment:
this used the wrong API (for comparing the field, which is not allowed when in nodes mode). Testing with debugAsserts enabled found this mistake.